lib/url.js

```diff
@@ -585,8 +585,8 @@ Url.prototype.format = function format() {
     host = auth + this.host;
   } else if (this.hostname) {
     host = auth + (this.hostname.indexOf(':') === -1 ?
-        this.hostname :
-        '[' + this.hostname + ']');
+      this.hostname :
```

Comment:
I think I was once asked to explain this snippet in a job interview 😵 
Maybe break after the open paran into an indented block:
```js
host = auth + (
  this.hostname.indexOf(':') === -1 ?
  this.hostname :
  '[' + this.hostname + ']'
);